OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -33,6 +33,9 @@ public static class LobbyUtils
 		[FluentReference]
 		const string Bots = "options-lobby-slot.bots";
 
+		[FluentReference]
+		const string UnnamedBot = "label-bot-name";
```

Comment:
IMO `UnnamedBot` isn't a good name for this. How about `GenericBotPlayer`?

Reply:
ofc, apparently I didn't push that change